screens/ProgramBuilder.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { createMachine } from "xstate";
+import {
+  fireEvent,
+  render,
+  renderApp,
+  screen,
+  waitFor,
+  within,
+} from "../tests/test.utils";
+import { createModel } from "@xstate/test";
+
+test("User goes to program builder screen from home", async () => {
+  const screen = renderApp();
+
+  const goToProgramBuilderButton = screen.getByText(/.*Program.*Builder.*/i);
+  fireEvent.press(goToProgramBuilderButton);
+  screen.findByTestId(`program-builder-screen-container`);
+});
+
+const programBuilderTestMachine = createMachine({
+  id: "program builder test machine",
+  initial: "Program builder screen view",
+  states: {
+    "Program builder screen view": {
```

Comment:
We could improve this by using nested state that will allow to verify the program builder screen presence each time :thinking: 
Indeed this could be written 100 ways, sticking to this for the moment